radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/p2p/PeerManager.java

```diff
@@ -178,6 +178,15 @@ public PeerManager(
         .findAny();
   }
 
+  public ImmutableSet<RadixNodeUri> activePeers() {
+    return this.activeChannels.values().stream()
+        .flatMap(Collection::stream)
+        .map(PeerChannel::getUri)
+        .filter(Optional::isPresent)
```

Comment:
This filters out any peers that are connected but we don't know their URI (for example incoming connections).
I think that in the ProxiedPeers message we should share a minimum of nodeId, host and port and only include URI if it's known. Perhaps we could reuse `PeerChannelInfo`?

Reply:
Done